Neos.Neos/Configuration/Caches.yaml

```diff
@@ -2,7 +2,7 @@ TYPO3_Neos_Configuration_Version:
   frontend: Neos\Cache\Frontend\StringFrontend
   backend: Neos\Cache\Backend\FileBackend
 
-TYPO3_Neos_TypoScript:
+Neos_Neos_Fusion:
```

Comment:
Yeah, i'll also add more commits to this one and change the other Cache key as well and provide a migration for all of them @kitsunet @robertlemke